README.md

```diff
@@ -13,6 +12,7 @@ The full repository name for Container Registry is `gcr.io/google.com/cloudsdkto
 
 ## Supported tags
 
+* `:stable`, `:VERSION-stable`: (default image with the basic version of gcloud without any pre-installed components, Debian-based)
```

Comment:
"with a standard gcloud installation, Debian-based"

Reply:
Updated the description for stable image in README.